src/main-api.ts

```diff
@@ -1,8 +1,18 @@
 import Vue from 'vue';
 import App from '@/views/App.vue';
+import { getMapConfig } from './map-config';
 
-new Vue({
-  render: (h) => h(App, {
-    props: { isBuiltForApi: true },
-  }),
-}).$mount('#app');
+async function startApp() {
+  const mapConfig = await getMapConfig();
```

Comment:
Can we add this also in the main-spa ? IMO this config can be useful in the main spa

Reply:
Sorry for the dumb question, what's a SPA?

Reply:
SPA is for Single Page Application, instead of having one htlm by page for a website, there is only one and all the job is done by JS but IDK how :man_shrugging:

I know that at the server level, you need a special configuration to [run a SPA](https://hub.docker.com/r/jawg/nginx-spa)

Reply:
Right, got it, I knew the term but not the acronym.
Maybe I'm getting old? ;(

Reply:
yah, SPA means that the server is pointing every request to the same html page, and javascript is doing the routing based on the url, not the hash

a bunch of hosting providers (google, netlify) have a one button click for SPA which is nice. github pages doesn't. We're also not running it in SPA mode anywhere at the moment, could entirely kill this file for now.

Reply:
Ha ha don't worry, as a backend developer, frontend things are always.... Complicated :laughing: